scripts/apps/archive/directives/MediaMetadata.ts

```diff
@@ -14,8 +14,12 @@ export function MediaMetadata(userList, archiveService, metadata, vocabularies)
             scope.$watch('item', reloadData);
             scope.isCorrectionWorkflowEnabled = appConfig?.corrections_workflow;
 
-            vocabularies.getVocabularies().then((vocabulariesCollection) => {
-                scope.label = (id) => getLabelForFieldId(id, vocabulariesCollection);
+            scope.label = (id) => id;
```

Comment:
It would be better to add a "loading" property which would be `true` until the real `scope.label` is set and to add `ng-if` in the template so `scope.label` would only be called after it was set. By doing that we would also avoid this dummy `scope.label` function which doesn't really resolve the label, but only returns the ID.